confluent_client/bin/nodersync

```diff
@@ -63,8 +65,16 @@ def run():
     c = client.Command()
     cmdstr = ' '.join(args[:-1])
     cmdstr = 'rsync -av --info=progress2 ' + cmdstr
-    if options.loginname:
-        cmdstr += ' {}@'.format(options.loginname) + '{node}:' + targpath
+    if options.loginname and options.substitutename: 
+        cmdstr += ' {}@'.format(options.loginname) + '{node}' + '{}:'.format(options.substitutename) + targpath
+    elif options.loginname and not options.substitutename:
+        cmdstr += ' {}@'.format(options.loginname) + '{node}:' + targpath  
+    elif options.substitutename and not options.loginname:
+        subname = options.substitutename
+        if '{' not in subname:
+            cmdstr += ' {node}' + '{}:'.format(subname)  + targpath
+        else:
+            cmdstr += ' {}:'.format(subname) + targpath
```

Comment:
I would normalize the 'substitutename' to be done in one sweep, e.g.:
targname = options.substitutename
if targname and '{' in targname:
    targname = '  ' + targname + ':'
elif targname:
    targname = '  {node}' + subname + ':'
else:
   targname = ' {node}:'

And then just use targname in the cmdstr, without having a lot of conditionals.